include/wasm/WasmModule.h

```diff
@@ -20,16 +23,19 @@
 
 #define WASM_BYTES_PER_PAGE 65536
 
+#define OPENMP_QUEUE_TIMEOUT_MS 10000
```

Comment:
This should only be hit when things have failed, it's to avoid an indefinite hanging when things break.